code/modules/mob/living/carbon/xenomorph/castes/baneling/abilities_baneling.dm

```diff
@@ -44,6 +44,15 @@
 	button.cut_overlay(visual_references[VREF_MUTABLE_BANE_CHARGES])
 	visual_references[VREF_MUTABLE_BANE_CHARGES] = null
 
+
+/datum/action/xeno_action/baneling_explode/can_use_action()
+	. = ..()
+	var/mob/living/carbon/xenomorph/X = owner
+	var/datum/action/xeno_action/spawn_pod/pod_action = X.actions_by_path[/datum/action/xeno_action/spawn_pod]
+	if(SSmonitor.gamestate == SHUTTERS_CLOSED && isnull(pod_action.the_pod))
+		X.balloon_alert(owner, span_notice("You can't explode before shutters if you don't have a pod!"))
```

Comment:
The message is a bit too long for balloon_alert, maybe change it to span_xenodanger or span_xenonotice?